voidcrew/code/modules/vending/boozeomat.dm

```diff
@@ -0,0 +1,6 @@
+/obj/machinery/vending/boozeomat/all_access/beach
+	all_items_free = TRUE
+	name = "\improper Super-Booze"
+	desc = "A Booze-O-Mat that appears to have its source of energy, as well as a completely free selection of items"
```

Comment:
```suggestion
	desc = "A conventiently placed Booze-O-Mat, seems to have free drinks!"
```
bike shedding but like im sure most people don't care about if it uses power, they will just see funny drink machine